st2actions/st2actions/runners/actionchainrunner.py

```diff
@@ -130,13 +130,16 @@ def pre_run(self):
             raise runnerexceptions.ActionRunnerPreRunError(message)
 
     def run(self, action_parameters):
-
         result = {'tasks': []}  # holds final result we store
         context_result = {}  # holds result which is used for the template context purposes
         top_level_error = None  # stores a reference to a top level error
         fail = True
         action_node = None
 
+        # Add action parameters to the context result so they can be accessed
+        # inside "publish"
+        context_result['parameters'] = action_parameters
```

Comment:
similar to https://github.com/StackStorm/st2/blob/action_chain_runner_publish_access_to_action_params/st2actions/st2actions/runners/actionchainrunner.py#L39 ?